packages/suite-desktop-core/webpack/core.rspack.config.ts

```diff
@@ -82,24 +82,25 @@ const config: webpack.Configuration = {
         ...dependencies,
         ...devDependencies,
         'bufferutil', // optional dependency of ws lib
-        'memcpy', // optional depencency of bytebuffer lib
```

Comment:
This seems outdated, because `memcpy` is never mentioned in any output bundle js file. The other externals are conditionally required there, but not `memcpy`..